tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -238,9 +249,29 @@ defmodule AstarteE2E.Config do
         ]
       }
     else
-      {:mail_adapter, {:ok, nil}} ->
+      _ ->
+        Logger.warn("Incomplete mail configuration. The Local Adapter will be used.",
+          tag: "local_adapter_fallback"
+        )
+
         %{chained_adapter: Bamboo.LocalAdapter}
```

Comment:
Call `fallback_config` here, otherwise you're building the fallback config in 3 different places